src/ipc/ipc-helper.c

```diff
@@ -40,7 +40,7 @@ struct comp_buffer *buffer_new(const struct sof_ipc_buffer *desc)
 {
 	struct comp_buffer *buffer;
 
-	tr_info(&buffer_tr, "buffer new size 0x%x id %d.%d flags 0x%x",
+	tr_info(&buffer_tr, "perf_mem buffer new size 0x%x id %d.%d flags 0x%x",
```

Comment:
We should put memory based profiling in another patch / PR.

Reply:
@btian1 We have heap debug separately and that is probably better place to track the memory usage.